resources/views/components/frameworks/bootstrap5/header/message-soft-deletes.blade.php

```diff
@@ -0,0 +1,9 @@
+@if(data_get($setUp, 'header.showMessageSoftDeletes') && ($softDeletes === 'withTrashed' || $softDeletes === 'onlyTrashed'))
+    <div class="alert alert-warning my-1" role="alert">
+        @if($softDeletes === 'withTrashed')
+            {{ trans('Exibindo todos registros, incluindo os excluídos.') }}
```

Comment:
Need to add insert translate for pt_Br and en for other translate

https://github.com/Power-Components/livewire-powergrid/blob/main/resources/lang/pt_BR/datatable.php

Example:
`@lang('livewire-powergrid::datatable.with-trashed.all')`